config/locales/account/fr.yml

```diff
@@ -12,6 +12,9 @@ fr:
       login: Information de connexion
       password: Mot de passe
       phone: Numéro de téléphone
+      piv_cac_card: Carte PIV/CAC
+      piv_cac_disabled: pas activé
+      piv_cac_enabled: activée
```

Comment:
The last `e` should be dropped.

Reply:
Or rather an `e` should be added to `pas activé` on line 16 since we're talking about a feminine `carte`.

Reply:
Even better, replace `pas activée` with `désactivée`

Reply:
Sounds good. I'm changing the Spanish to mirror deactivated/activated.